docs/resources/cbr_vault.md

```diff
@@ -0,0 +1,190 @@
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the CBR vault.
+  If omitted, the provider-level region will be used.
+  Changing this will create a new vault.
+
+* `name` - (Required, String) Specifies a unique name of the CBR vault.
+  This parameter can contain a maximum of 64 characters, which may consist of letters, digits,
+  underscores(_) and hyphens (-).
+
+* `type` - (Required, String, ForceNew) Specifies the object type of the CBR vault.
+  Vaild values are: server (Cloud servers), disk (EVS disks) and turbo (SFS turbo file systems).
+  Changing this will create a new vault.
+
+* `consistent_level` - (Required, String, ForceNew) Specifies the backup specifications.
+  The valid values are *[crash_consistent](https://support.huaweicloud.com/intl/en-us/usermanual-cbr/cbr_03_0109.html)*
+  and *[app_consistent](https://support.huaweicloud.com/intl/en-us/usermanual-cbr/cbr_03_0109.html)*
+  (backup backs up memory data of databases and other applications on a __server__ to ensure application consistency).
+  default to crash_consistent. Changing this will create a new vault.
+
+* `protection_type` - (Required, String, ForceNew) Specifies the protection type of the CBR vault.
+  The valid values are backup and replication. Vaults of type disk don't support replication.
+  Changing this will create a new vault.
+
+* `size` - (Required, Int) Specifies the vault sapacity, in GB. The valid value is ranges from 1 to 10485760.
+
+* `auto_expand` - (Optional, Bool) Specifies to enable auto capacity expansion for the disk vault with replicate
+  protection type. Default to false. `auto_expand` ture means if the storage capacity of a disk vault is used up,
+  the disk vault capacity will be automatically expanded to 1.25 times the storage capacity of the original vault.
+
+* `enterprise_project_id` - (Optional, String, ForceNew) Specifies a unique ID in UUID format of enterprise project.
+  Changing this will create a new vault.
+
+* `policy_id` - (Optional, String) Specifies a policy to associate with the CBR vault.
+  `policy_id` cannot be used with the vault of replicate protection type.
+
+* `resources` - (Optional, List) Specifies an array of one or more resources to attach to the CBR vault.
+  The resources structure is documented below.
+
+The `resources` block supports:
+
+  * `id` - (Required, String) Specifies the ID of the resource to be backed up.
+
+  * `exclude_volumes` - (Optional, List) Specifies the disks included in the backup except for the array that contains
+  one or more disk IDs.
+
+  * `include_volumes` - (Optional, List) Specifies the array of disk IDs which will be included in the backup.
+  This parameter and `exclude_volumes` are alternative.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - A resource ID in UUID format.
+
+* `allocated` - The allocated capacity of the vault, in MB.
```

Comment:
please convert the unit to GB

Reply:
thanks, the unit is update to __GB__ and the type of allocated is update to float64 (base for TypeFloat).